Runtime/Scripts/MeshSyncServer.Properties.cs

```diff
@@ -262,5 +267,11 @@ protected override void SplineChanged(Spline spline, int arg2, SplineModificatio
         needsClientSync = true;
     }
 #endif
+
+
+        internal void RequestUserScriptCallback() {
+            needsUserScriptCallback = true;
+        }
+#endif
```

Comment:
where is the opening directive? I think this is causing compilation errors in yamato, all test are failing